src/CHANGES.txt

```diff
@@ -6,9 +6,11 @@
 
 RELEASE  VERSION/DATE TO BE FILLED IN LATER
 
-  From John Doe:
+  From Mathew Robinson:
 
-    - Whatever John Doe did.
+    - Improved threading performance by ensuring NodeInfo is shared
```

Comment:
Hmm. maybe add for Parallel builds (using -j # > 1) or something to indicate when this applies?

Reply:
Done.